ansible/roles/docker/tasks/debian.yml

```diff
@@ -28,6 +28,8 @@
   retries: 5
   delay: 5
 
+# Required by the docker_image ansible module
```

Comment:
This looks like a bug. If this is also in the main.yml file, we shouldn't need to redo it for debian specifically.